### PR TITLE
Gradle updates

### PR DESCRIPTION
**Pull request by CodeCombiner**

### CODE REVIEW

**Comment by rampr** [comment]
Alex, we would like at add custom styling like we did in #13 
How can it be done with these changes?

**Comment by rampr** [comment]
Should I keep it open until custom styling in supported or can it be a separate PR ?

**Comment by CodeCombiner** [comment]
I think separate is better

**Comment by rampr** [comment]
understood, I will merge this then

File: build/index.js [hunk 1]
```diff
@@ -12,7 +12,12 @@ exports.onCreateProject = function (api, app, config, cb) {
   if (config.target == 'native-android') {
     xmlStr = new xmldoc.XmlDocument(fs.readFileSync(path.join(__dirname, '../android/strings.xml'), 'utf-8'));
     xmlStr.children[0].val = packageName + ".provider";
-    return fs.outputFileAsync(path.join(androidProjectPath, 'res/values', strings_file), xmlStr.toString(), 'utf-8');
+      return fs.outputFileAsync(path.join(androidProjectPath,'../../',
```
Comment (rampr) [inline build/index.js:15]: indentation - has additional 2 spaces

**Review by vivekb91 — APPROVED** [review]

File: build/index.js (hunk 1 above)
Comment (rampr) [reply build/index.js:15]: ignoring this for now.

**Review by rampr — APPROVED** [review]

**Pull request merged**